packages/liveblocks-client/src/AuthToken.ts

```diff
@@ -135,10 +136,13 @@ export function parseRoomAuthToken(
     } = data;
     return token;
   } else {
-    throw new Error("Authentication error: invalid room auth token");
+    throw new Error(
+      "Authentication error: we expected a room token but did not get one. Hint: if you are using a callback, ensure the room is passed when creating the token. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClientWithCallback"
```

Comment:
I noticed the hash was incorrect here?

```suggestion
      "Authentication error: we expected a room token but did not get one. Hint: if you are using a callback, ensure the room is passed when creating the token. For more information: https://liveblocks.io/docs/api-reference/liveblocks-client#createClientCallback"
```

Reply:
Also perhaps say "room ID" in the error message, not just "room"? This might also be something to mention in those linked docs, because it's also not clear from the docs what exactly that `room` param is that the callback receives: a string? or an object?

Reply:
Indeed, thanks for the 🦅 eyes! 

Will adjust both, good catches 😄